src/plugins/copyPaste/copyPaste.js

```diff
@@ -125,15 +131,18 @@ class CopyPaste extends BasePlugin {
     if (this.enabled) {
       return;
     }
-    const settings = this.hot.getSettings();
 
-    this.#isFragmentSelectionEnabled = settings.fragmentSelection;
+    const { copyPaste: settings, fragmentSelection } = this.hot.getSettings();
```

Comment:
Renaming `copyPaste` to `settings` is confusing. My first thought was that `pasteMode` is not in our main settings object.

Reply:
This is not my code, it is on `develop` branch.
As I said above, `copyPaste` is only for only demonstrative purpose. And I'll remove all changes after review.

Reply:
Oops, sorry. Didn't noticed that. I had a highlighted change but you're right, that was `develop` merge.

I don't think you should remove the changes in `copyPaste`. New operators should stay as a part of the code base and be a living proof that the babel plugins are working correctly.

Reply:
I think that you can remove all code not related to this PR before review. The description of this PR is enough descriptive to handle a review. Moreover, I can look into the code with example syntax by analyzing previous commits.

Reply:
Right, I'm not the reviewer 🤐 😄 👍